### DISCUSSION TITLE
I'm trying to apply for the Github Student Developer Pack, but i can't upload because of github error

### DISCUSSION THREAD

**Discussion by suku1515** [community/community#58825] [GitHub Education]
### Hi everyone,

I sent a certificate of enrollment in English to authenticate as a Github student. However, I couldn't solve it because of two problems and an error in Github. The reason is: You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

However, here, looking at the picture, the original school name is MokpoYudal Middle School (mokpoyudal.ms.jne.kr), but it is clearly different from Mokpo High School, so it is not certified. What should I do? Please help me
![247870521-6275f0f9-0d89-45e3-b137-0be3a9d166b6](https://github.com/community/community/assets/136946245/48c4d38d-81b9-401f-8329-f065206bbdae)

**Comment by samus-aran** [comment]
Hi @suku1515  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
